packages/web/utils/menuLinks.ts

```diff
@@ -19,14 +41,14 @@ export const MenuSectionLinks: MenuLinkSet[] = [
         title: 'Players',
         explainerText:
           'Find players of MetaGame; their NFTs, their skills & whatever else they put on there',
```

Comment:
Why are some descriptions inline and the rest above?

Reply:
I put some descriptions above, because I can use them in the `HeaderComponent`, which seems to be used for SEO:
```
<HeadComponent
      title={`Events`}
      description={descriptions.events}
      url={`https://wiki.metagame.wtf/docs/great-houses/house-of-daos`}
    />
```